app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -190,16 +190,9 @@ function clearAdjustmentReason() {
 }
 
 function renderTagOnArticle(tagName, colors) {
-  /* eslint-disable no-restricted-globals */
-  let articleTagsContainer;
-  if (top.document.location.pathname.endsWith('/mod')) {
-    [articleTagsContainer] = top.document
-      .getElementById('quick-mod-article')
-      .contentDocument.getElementsByClassName('tags');
-  } else {
-    [articleTagsContainer] = top.document.getElementsByClassName('spec__tags');
-  }
-  /* eslint-enable no-restricted-globals */
+  const [articleTagsContainer] = window.parent.document
+    .getElementsByClassName('article-iframe')[0]
+    .contentWindow.document.getElementsByClassName('spec__tags');
```

Comment:
`top.document.getElement...` doesn't work when testing in Cypress (since the Cypress test runner iframe is `top`), but `window.parent` works for both contexts